package.json

```diff
@@ -124,7 +124,7 @@
     "@storybook/react": "^6.4.21",
     "@tailwindcss/forms": "^0.5.0",
     "@testing-library/jest-dom": "^5.16.3",
-    "@testing-library/react": "^12.1.4",
+    "@testing-library/react": "^13.3.0",
```

Comment:
Upgrading this fixed the `createRoot` vs `render` warning and our tests all still passed so...